NetCore/LoadResizeSave.cs

```diff
@@ -143,6 +145,33 @@ public void SystemDrawingResize(string input)
             }
         }
 
+        [Benchmark(Description = "ImageFlow Load, Resize, Save"/*,
+            OperationsPerInvoke = ImagesCount*/)]
+        public void ImageFlowBenchmark()
+        {
+            foreach (string image in Images)
+            {
+                ImageFlowResize(image);
+            }
+        }
+
+        public void ImageFlowResize(string input)
+        {
+            using (var output = File.Open(OutputPath(input, ImageFlow), FileMode.Create))
+            {
+                var imageBytes = File.ReadAllBytes(input);
+                // Resize it to fit a 150x150 square
+                using (var image = new ImageFlow.ImageJob())
```

Comment:
This doesn't seem to be outputting any information.

<img alt="image" src="https://user-images.githubusercontent.com/385879/198821712-41a1a29e-c2fb-4f12-8fba-88f9ca6b25dd.png">

Reply:
Haha - well that's embarrassing, I only saw it created the file didn't actually open it. Should be fixed now though !